airbyte-integrations/bases/bases-destination-jdbc/src/main/java/io/airbyte/integrations/destination/staging/AsyncFlush.java

```diff
@@ -95,7 +94,7 @@ public void flush(final StreamDescriptor decs, final Stream<PartialAirbyteMessag
   @Override
   public long getOptimalBatchSizeBytes() {
     // todo(davin): this should be per-destination specific. currently this is for Snowflake.
-    return 200 * 1024 * 1024;
+    return 100 * 1024 * 1024;
```

Comment:
By lowering the `optimalBatchSize` we can achieve about 40% speed improvement in a single stream with 1Gi memory on the container. This is because each async `copyIntoTable` operation takes about 9 seconds and by creating smaller chunks we're able to use a sliding window to free up memory earlier